people/time-off.md

```diff
@@ -44,6 +44,25 @@ Follow the process described above in this case - you are welcome to describe yo
 In this case, our guidelines around letting others know ahead of time are somewhat relaxed, as sick leave may happen unexpectedly.
 Just do the best you can, and try to balance your own constraints with respect for the team’s bandwidth and planning.
 
+(time-off:annual-expected)=
+## End-of-year expected time off
+
+The **final 14 days and the first 3 days of the year are "expected time off" for all of 2i2c**.
+
+During this time our organization operates at a reduced capacity and with greatly diminished commitments to our {external+docs:ref}`service level objectives <objectives:stability>`.
+
+Our support and operational roles are still staffed, but with greatly reduced expectations around responsiveness and resolution.
+Team members can use this time as they wish (including continuing to work), but we wish to set the _expectation_ with internal and external partners that we will be less responsive.
+
+Here is our policy during this time:
+
+- All team meetings are cancelled during this time.
+- We monitor our support e-mail for major incidents, but do not guarantee a response for non-essential requests or questions.
+- If there is an incident, we will take steps to minimize damage and harm, and will plan time _after this period_ to resolve underlying problems.
+- We do not perform any non-essential cloud changes for communities.
+
+See [](support:expected-time-off) for our support changes during this time.
```

Comment:
I understand the following about the responsibility of a support steward during these expected reduced availability periods:

*The support steward will monitor the support email during this time and will react to incidents by doing their best to ensure hub is up and available*

Did I understand the list correctly?

Reply:
No, I think during this period, the support steward should do their best to make sure the hub is "safe", by which I mean there's no cryptominers, etc. We should divert efforts to make sure the hub is up and available and fixing issues for the long-term to when the whole team are back

Reply:
I think the Engineering team should decide what kind of promises they'd like to make, I think our two choices are basically:

- We'll only provide fixes for catastrophic problems (e.g. crypto mining burning through credits) but won't treat "a general outage" as catastrophic unless we get an explicit signal from a community representative. OR:
- We'll keep hubs up-and-running during expected downtime.

Reply:
I think we should aim for the first point. Most of our communities are in the global north and will therefore be in a similar reduced capacity too, so I definitely think a hub outage during this period is not catastrophic.